plugins/woocommerce/tests/e2e-pw/tests/js-file-monitor/monitor-js-file-number.spec.js

```diff
@@ -50,12 +50,12 @@ test.describe( 'Keeps track of the number of JS files included on key shopper pa
 	for ( const row of shopperPages ) {
 		const url = row.url;
 		const name = row.name;
-		const expectedCount = parseInt( row.expectedCount );
+		const expectedCount = parseInt( row.expectedCount, 10 );
 
 		test( `Check that ${ name } has ${ expectedCount } JS files`, async ( {
 			page,
 		} ) => {
-			await page.goto( url, { waitUntil: 'networkidle' } );
```

Comment:
The default event is `load`, which waits for the whole page to be loaded, including all dependent resources.